how-to/setup-an-example-websocket-server/public/index.js

```diff
@@ -0,0 +1,363 @@
+            "buttons": [
+                {
+                    "onClick": {
+                        "task": "broadcast",
+                        "customData": {
+                            "id": "custom-app-channel",
+                            "context": {
+                                "type": "fdc3.contact",
+                                "name": "John Example",
+                                "id": {
+                                    "email": "john@example.com",
+                                    "phone": "Number goes here"
+                                }
+                            }
+                        }
+                    },
+                    "cta": true,
+                    "title": "Broadcast On App Channel",
+                    "type": "button"
+                }
+            ]
+    }
+};
+
+// endpoint notification
+sampleMessages["notificationActionEndpoint"] = {
+    "eventId": "create",
+    "payload": {
+            "id": "guid-goes-here",
+            "title": "Trigger a post to a backend through a CTA",
+            "body": "Click the button to call an action on the specified endpoint.",
+            "buttons": [
+                {
+                    "onClick": {
+                        "task": "endpoint",
+                        "customData": {
+                            "id": "endpointId you wish to call. The notification service module must be permitted to access this endpoint through endpointClients",
+                            "context": {
+                                "type": "fdc3.contact",
+                                "name": "John Example",
+                                "id": {
+                                    "email": "john@example.com",
+                                    "phone": "Number goes here"
+                                }
+                            },
+                            "endpointOptions": {
+                                "request": {
+                                    "description": "If request is specified then context will be ignored and this object will be sent to the endpoint as the request"
+                                }
+                            }
+                        }
+                    },
+                    "cta": true,
+                    "title": "Send to Endpoint",
+                    "type": "button"
+                }
+            ]
+        }
+};
+
+// update notification
+sampleMessages["notificationUpdate"] = {
+    "eventId": "update",
+    "payload": {
+            "id": "id-of-notification-to-update",
+            "body": "notification has been updated and buttons have been removed.",
+            "template": "markdown"
+        }
+};
+
+// notification to clear
+sampleMessages["notificationClear"] = {
+    "eventId": "clear",
+    "payload": {
+            "id": "id-of-notification-to-clear"
+    }
+};
+
+/**
+ * Add some logging.
```

Comment:
Comment mismatch

Reply:
Fixed. I also broke out messages into their own file to make it easier to manage messages, make the js simpler and allow for easy extension.